server/resourcemanager/operations.go

```diff
@@ -0,0 +1,163 @@
+package resourcemanager
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kubeshop/tracetest/server/id"
+	"golang.org/x/exp/slices"
+)
+
+type Operation string
+
+const (
+	OperationList   Operation = "list"
+	OperationCreate Operation = "create"
+	OperationUpdate Operation = "update"
+	OperationGet    Operation = "get"
+	OperationDelete Operation = "delete"
+)
+
+var availableOperations = []Operation{
+	OperationList,
+	OperationCreate,
+	OperationUpdate,
+	OperationGet,
+	OperationDelete,
+}
+
+type SortableHandler interface {
+	SortingFields() []string
+}
+
+type List[T ResourceSpec] interface {
+	SortableHandler
```

Comment:
This looks a little bit complex to me, why do we need this file?

Reply:
We need to be able to selectivly enable endpoints. the code in this file enables that behaviour